sortedm2m_tests/models.py

```diff
@@ -3,8 +3,14 @@
 from sortedm2m.fields import SortedManyToManyField
 
 
+class m2mprint:
+
+    def __unicode__(self):
+        return unicode(self.book)
```

Comment:
This throws and error saying the maximum recursion depth has exceeded. Which IMO, is maybe because `str(self)` is not invoking the `__str__` method. It is perhaps calling the `__unicode__` method. Idk, if this is the problem with python version or something. @nemesisdesign